types/react-transition-group/react-transition-group-tests.tsx

```diff
@@ -17,6 +17,7 @@ const Test: React.StatelessComponent = () => {
         <TransitionGroup
```

Comment:
`import * as X from 'y';` is an ES6 module object import.
ES6 module object imports are never callable/constructable.
You are attempting to call or construct an ES6 module object import, which is not legal.

Whatever transpiler set up you have that makes this happen to work is irrelevant. `import * as X from 'y';` followed by `X()` or `new X()` is always wrong and attempting to make it work by breaking the definition file is just going to break people who are doing correct things. This change will not be merged